Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2_test.py

```diff
@@ -0,0 +1,116 @@
+import json
+import pytest
+from stix2 import TAXIICollectionSource
+from test_data.mitre_test_data import ATTACK_PATTERN, COURSE_OF_ACTION, INTRUSION_SET, MALWARE, TOOL, ID_TO_NAME, \
+    RELATION
+
+
+def mock_create_relations(original):
+    def mock(item_json, id_to_name):
+        return original(item_json, ID_TO_NAME)
+    return mock
+
+
+@pytest.mark.parametrize('indicator, expected_result', [
+    # ([ATTACK_PATTERN.get('response')], ATTACK_PATTERN.get('indicator')),
+    # ([COURSE_OF_ACTION.get('response')], COURSE_OF_ACTION.get('indicator')),
+    # ([INTRUSION_SET.get('response')], INTRUSION_SET.get('indicator')),
+    # ([MALWARE.get('response')], MALWARE.get('indicator')),
+    # ([TOOL.get('response')], TOOL.get('indicator')),
+    ([RELATION.get('response')], RELATION.get('indicator'))
+])
+def test_fetch_indicators(mocker, indicator, expected_result):
+    """
+    Given
+    - fetch incidents command
+    - command args
+    - command raw response
+    When
+    - mock the Client's get_stix_objects.
+    Then
+    - run the fetch incidents command using the Client
+    Validate that all the indicators extracted successfully
+    """
+    import FeedMitreAttackv2 as fm
+    from FeedMitreAttackv2 import Client, create_relationship
+    client = Client(url="https://cti-taxii.mitre.org", proxies=False, verify=False, tags=[], tlp_color=None)
```

Comment:
is this actually connecting to chi-taxii?

Reply:
yes